src/frontend/web_application/src/components/ContactDetails/index.jsx

```diff
@@ -74,22 +87,22 @@ class ContactDetails extends Component {
           && remoteIdentity.identity_id === identityId);
   }
 
-  renderSubtitleActions() {
+  renderSubtitleActions(panel) {
     const { __ } = this.props;
-    const activeButtonProp = this.state.editMode ? { color: 'active' } : {};
+    const activeButtonProp = this.state.editMode[panel] ? { color: 'active' } : {};
 
     return (
       <Button
```

Comment:
ok I understand ... so it's sure, we can't use "value" property in the button, it's not its purpose (since there is no props `value` in Button component).

I recently got what you wanted to do with [currying](https://medium.com/@kbrainwave/currying-in-javascript-ce6da2d324fe) so "value" is no more required:

```diff
...
- handleSwitchEditMode() {
+ makeHandleSwitchEditMode(panel) {
+   return () => {
      this.setState(prevState => ({
-       editMode: !prevState.editMode,
+       editMode: {
+         [panel]: !prevState.editMode[panel],
+       }
     }));
+  };
}
...
- onClick={this.handleSwitchEditMode}
+ onClick={this.makeHandleSwitchEditMode(panel)}
```

(actually it's not currying as described in the article but it's the same result)

I hope this fix your issue